account_analytic_distribution/models/account_invoice.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# Copyright 2017 Vicent Cubells - <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    def inv_line_characteristic_hashcode(self, invoice_line):
+        code = super(AccountInvoice, self).inv_line_characteristic_hashcode(
```

Comment:
super can be simplified with the form `super()`

https://docs.python.org/3.0/library/functions.html#super